fitbenchmarking/fitbenchmarking
# Fixing table software links

#### Description of Work

Fixes #630 

Fixes software links in the tables

#### Testing Instructions

1. Check all the links work as expected

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
